packages/pretty-format/src/index.js

```diff
@@ -397,20 +397,44 @@ const DEFAULTS: Options = {
 function validateOptions(opts: InitialOptions) {
   Object.keys(opts).forEach(key => {
     if (!DEFAULTS.hasOwnProperty(key)) {
-      throw new Error('prettyFormat: Invalid option: ' + key);
+      throw new Error(`pretty-format: Unknown option "${key}"`);
     }
   });
 
   if (opts.min && opts.indent !== undefined && opts.indent !== 0) {
-    throw new Error('prettyFormat: Cannot run with min option and indent');
+    throw new Error('pretty-format: Options "min" and "indent" cannot be used together');
   }
 }
 
+function normalizeTheme(themeOption: mixed) {
+  if (themeOption === null) {
+    throw new Error(`pretty-format: Option "theme" must not be null`);
+  }
+  if (typeof themeOption !== 'object') {
+    throw new Error(`pretty-format: Option "theme" must be of type object but instead received ${typeof themeOption}`);
+  }
+
+  const themeDefaults = DEFAULTS.theme;
+  return Object.keys(themeDefaults).reduce((theme, key) => {
+    // $FlowFixMe Method cannot be called on mixed
+    theme[key] = themeOption.hasOwnProperty(key)
```

Comment:
Let's do `Object.prototype.hasOwnProperty.call(themeOption, key)`. That should get rid of the flow error.